pkg/pgctl/pgrestore.go

```diff
@@ -76,3 +107,25 @@ func (x *Restore) SetOptions(o []string) {
 func (x *Restore) GetOptions() []string {
 	return x.Options
 }
+
+// DropSchemas drops all schemas except the system ones.
+func (x *Restore) DropSchemas() error {
```

Comment:
The pattern of the other methods is to return a Result. Maybe that's a better option so stdout/stderr of the command can be handled more strictly.
```suggestion
func (x *Restore) DropSchemas() Result {
```

Reply:
Changed